src/site/content/en/fast/efficiently-load-third-party-javascript/index.md

```diff
@@ -0,0 +1,174 @@
+
+{% endAside %}
+
+### `dns-prefetch`
+
+`<link rel="dns-prefetch>` handles a small subset of what is handled by `<link rel="preconnect">`.  Establishing a connection involves the DNS lookup and TCP handshake, and for secure origins, TLS negotiations. `dns-prefetch` instructs the browser to only resolve the DNS of a specific domain before it has been explicitly called.
+
+Because there’s a limit to the number of connections a browser can open, for less critical third-party origins use `<link rel=dns-prefetch>`. 
+
+```html
+<link rel="dns-prefetch" href="http://example.com">
+```
+
+[Browser support for `dns-prefetch`](https://caniuse.com/#search=dns-prefetch) is slightly different from [`preconnect` support](https://caniuse.com/#search=preconnect), so `dns-prefetch` can serve as a fallback for browsers that don’t support `preconnect`. Use separate link tags to implement this safely: 
+
+```html
+<link rel="preconnect" href="http://example.com">
+<link rel="dns-prefetch" href="http://example.com">
+```
+## Lazy-load third-party resources
+
+Embedded third-party resources can be a big contributor to slow page speed when constructed poorly. If they aren’t critical or are below the fold (that is, if users have to scroll to view them), lazy-loading is a good way to improve page speed and paint metrics. This way users will get the main page content faster and have a better experience.
+
+<figure class='w-figure w-figure--inline-left'>
+<img src="lazy-load.png" alt="A diagram of a webpage shown on a mobile device with scrollable content extending beyond the screen. The content that's below-the-fold is desaturated because it's not loaded yet.">
+</figure>
+
+One way to do that is to lazy-load third-party content after the main page content loads. Ads are a good candidate for this approach.
+
+Ads are an important source of income for many sites, but users come for the content. By lazy-loading ads and delivering the main content faster, you can increase the overall viewability percentage of an ad. For example, MediaVine switched to [lazy-loading ads](https://www.mediavine.com/lazy-loading-ads-mediavine-ads-load-200-faster/) and saw a 200% improvement in page load speed. DoubleClick have guidance on how to lazy-load ads in their [official documentation](https://support.google.com/dfp_premium/answer/4578089#lazyloading). 
+
+An alternative approach is to load third-party content only when users scroll down to that section of the page.
+
+[Intersection Observer](https://developers.google.com/web/updates/2016/04/intersectionobserver) is a browser API that efficiently detects when an element enters or exits the browser's viewport and it can be used to implement this technique.
```

Comment:
Should this and the following paragraph be attached to the preceding paragraph--to make it clearer that they're all about lazy-loading based on scroll position?